manifests/monitoring/alertmanager-rules.yaml.raw

```diff
@@ -861,3 +866,22 @@ spec:
           for: 10m
           labels:
             severity: warning
+    - name: pod-resource-allocation
+      rules:
+        - alert: ContainerOverProvisioned
+          annotations:
+            message: Container {{ $labels.container }} in Pod {{ $labels.pod }} is using less than 10 percent of allocated CPU
+          expr: |
+            namespace_pod_container:container_cpu_usage_percentage:sum_rate < 10 and
+            (sum by (container, pod, namespace) (kube_pod_container_resource_requests_cpu_cores{namespace!="postgres-operator"})) > .2
+          for: 10m
+          labels:
+            severity: warning
+        - alert: ContainerRequestTooLarge
+          annotations:
+            message: Container {{ $labels.container }} in Pod {{ $labels.pod }} is requestion more than 50 percent of scheduled node
```

Comment:
Not sure if the labels.node will work, but I think it will provide a clearer message

```suggestion
            message: Container {{ $labels.container }} in Pod {{ $labels.pod }} is requesting more than 50% of {{ $labels.node}}
```